samples/graphql/graph/status.graphqls

```diff
@@ -0,0 +1,23 @@
+# GraphQL schema example
+#
+# https://gqlgen.com/getting-started/
+
+type Status {
+	State: String!
+  Redis: String!
+  Uptime: String!
```

Comment:
Weird indentation 🤔 
Do you have installed GraphQL extension?

Reply:
Yes I have! It seems fine in the editor. I will check it out why it shows up like this here.